lib/board.rb

```diff
@@ -32,8 +37,45 @@ def free_positions
     flat_board.each_index.select { |index| empty?(flat_board[index]) }
   end
 
+  def content
```

Comment:
Github confuse me. If you are talking about the free_positions methods it's used by the computer to know which move it can attempt and for the verification a valid move by a player


board.free_positions.include?(next_move_position)